Dockerfile.maven

```diff
@@ -0,0 +1,4 @@
+FROM ghcr.io/dependabot/dependabot-updater-core
+
+USER dependabot
+COPY --chown=dependabot:dependabot maven /home/dependabot/maven
```

Comment:
Since there's nothing customized in here, could this be parameterized as a basic dockerfile that gets used in any ecosystem that doesn't need customization?

Reply:
Possibly, but I would like to not refactor as we break things out to avoid extra risk.